apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2

```diff
@@ -56,6 +56,8 @@ spec:
                 secretKeyRef:
                   name: newrelic-secrets
                   key: NEW_RELIC_LICENSE_KEY
```

Comment:
I forgot to mention that I wrote the ssr server to use new relic if NEW_RELIC_LICENSE_KEY and NEW_RELIC_APP_NAME are set, and I suspect that the logging information we get would be really valuable if we set them in the ssr.deploy.yaml.j2 file.  (not sure actually where the _APP_NAME gets set)